mlflow/langchain/langchain_tracer.py

```diff
@@ -29,9 +29,24 @@ class MlflowLangchainTracer(BaseCallbackHandler, metaclass=ExceptionSafeAbstract
     Callback for auto-logging traces.
     We need to inherit ExceptionSafeAbstractClass to avoid invalid new
     input arguments added to original function call.
+    Args:
```

Comment:
```suggestion

    Args:
```

Reply:
Do you use any auto-formatter for docstrings?

Reply:
I don't :)

Reply:
@serena-ruan Filed https://github.com/mlflow/mlflow/pull/11604